var/spack/repos/builtin/packages/r-cli/package.py

```diff
@@ -34,7 +35,7 @@ class RCli(RPackage):
     version("1.0.0", sha256="8fa3dbfc954ca61b8510f767ede9e8a365dac2ef95fe87c715a0f37d721b5a1d")
 
     depends_on("r@2.10:", type=("build", "run"))
-    depends_on("r@3,4:", type=("build", "run"), when="@3.3.0:")
```

Comment:
This was a typo in the spack repo; ref https://github.com/cran/cli/blob/79876598df96964dee9032362b0773868d624021/DESCRIPTION#L21.